tests/integration_test.go

```diff
@@ -693,6 +694,7 @@ func (s *IntegrationTestSuite) testAbortXMRMakerCancels(asset types.EthAsset) {
 	default:
 	}
 
+	common.SleepWithContext(ctx, 2*time.Second) // give some extra time for the offer to be re-added
```

Comment:
Our `TestAbort_ERC20_XMRMakerCancels` integration test is flaky. Interestingly, I'm only seeing the ERC20 version of this test fail sporadically, not the mostly-shared-code `TestAbort_XMRMakerCancels` test. On line 700 below, the length is zero when it is supposed to be one. Do you agree that there is a delay between the maker exiting his go process above and the offer being re-added? Since the failure is sporadic, I can't be certain this fix is helping.

Reply:
yeah, it's probably a timing issue, does increasing this timeout help?

Reply:
I think so.